package.json

```diff
@@ -73,6 +74,7 @@
     "@bjeco/prettier-config": "0.1.6",
     "@changesets/cli": "2.17.0",
     "@espressive/eslint-config": "workspace:*",
+    "@espressive/prop-types": "workspace:*",
```

Comment:
This dependency is not used at the root of our monorepo. The only dependencies that should be declared at the root are typically those that are being used at build time. I see there are others here that should not be present as well like fluent-ui, which is probably confusing.

Go ahead and leave this out for now at the root. It should only be present in packages that use it. For instance... it should be included as a devDependency for `@espressive/cascara`